app/assets/stylesheets/modules/typography.scss

```diff
@@ -0,0 +1,31 @@
+@import "foundation_and_overrides";
+@import "globals/all";
+
+$text-column-counts: 2 3 4 5 6;
+
+@each $text-column-count in $text-column-counts {
+
+  .text-columns-#{$text-column-count} {
+    column-count: $text-column-count;
```

Comment:
Properties should be ordered -moz-column-count, -webkit-column-count, column-count